flexget/plugins/filter/unique.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('unique')
+
+
+class Unique(object):
+    """
+    Take action on entries with duplicate fields, except for the first item
+
+    Reject the second+ instance of every movie:
+
+      unique:
+        field:
+          - imdb_id
+          - movie_name
+        action: reject
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'field': {
+                'oneOf': [
+                    {'type': 'string'},
+                    {'type': 'array'},
+                ]
+            },
+            'action': {'enum': ['accept', 'reject']},
+        },
+        'required': ['field'],
+        'additionalProperties': False
+    }
+
+    def prepare_config(self, config):
+        if isinstance(config, bool) or config is None:
+            config = {}
+        config.setdefault('action', 'reject')
+        if not isinstance(config['field'], list):
+            config['field'] = [config['field']]
+        return config
+
+    def extract_fields(self, entry, field_names):
+        return [ entry.get(field) for field in field_names ]
```

Comment:
I could, but if that's what the user wants to determine uniqueness by...
I'd prefer that than failing the run because of a missing field. Not sure what's the flexget norms on that